sql/core/src/test/resources/sql-tests/results/ansi/literals.sql.out

```diff
@@ -213,9 +219,15 @@ struct<0.3:decimal(1,1),-0.8:decimal(1,1),0.5:decimal(1,1),-0.18:decimal(2,2),0.
 -- !query 22
 select 123456789012345678901234567890123456789e10d, 123456789012345678901234567890123456789.1e10d
 -- !query 22 schema
-struct<1.2345678901234568E48:double,1.2345678901234568E48:double>
+struct<>
 -- !query 22 output
-1.2345678901234568E48	1.2345678901234568E48
```

Comment:
so the previous result was wrong?

Reply:
No, I just realize that a bug has been introduced in `numericLiteral()` if we simply use `contains(E)` to recognize exponent value. Because we also have `DIGIT+ EXPONENT? 'D'`, e.g. 10E2D, for double, which results in collision.